diario/main.py

```diff
@@ -67,7 +73,28 @@ def post_todays_gazette(gazettes: list):
             f"de #FeiradeSantana ({gazette['date']} - {gazette['year_and_edition']}). "
             f"📰\n{gazette['files'][0]['url']}"
         )
-        tweet(tweet_message)
+        tweet_id = tweet(tweet_message)
+        logger.info("The gazette was posted on twitter!")
+
+        keywords = json.loads(os.getenv("KEYWORDS", "{}"))
+        events_text = "".join(event["summary"] for event in gazette["events"])
+
+        found_topics = extract_keywords(events_text, keywords)
+        if found_topics:
+            character_number = sum([len(topic_len) for topic_len in found_topics])
+            if character_number > CHARACTER_LIMIT:
+                tweets = split_tweets(found_topics, CHARACTER_LIMIT)
+                for post in tweets:
+                    if tweets[0] == post:
+                        reply_message = f"Nele temos: {', '.join(post)}"
+                    else:
+                        reply_message = f"Temos também: {', '.join(post)}"
+                    tweet_id = tweet(reply_message, tweet_id)
+                    logger.info("The thread was posted")
+            else:
+                reply_message = f"Nele temos: {', '.join(found_topics)}"
+                tweet_id = tweet(reply_message, tweet_id)
```

Comment:
```suggestion
                tweet(reply_message, tweet_id)
```
Não precisamos dessa variável aqui porque ela não será usada depois.